### PR TITLE
Register earth.is-a.dev

### PR DESCRIPTION
**Pull request by sIumberdemon**
Register earth.is-a.dev with CNAME record pointing to imdakiki.github.io/earth.

### CODE REVIEW

File: domains/earth.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "imdakiki",
+           "email": "",
+           "discord": "898255050592366642",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.IaEjpjC1tqI3LaIBPKNHiotfKngSRpLBMGFsV3dg18jGSzG2RoF4lLTrY3f0ZUXcTE-ih1NxoiFAXqlw4BXHVnJ2SVHD12U3LqBd5OWjcEm5LqljME2JnXhsu0bHIf2iwSDiR61D6IH4w1_PDoNYwbLc8qCDqp4vayZJMyTrN_TXH2dodU-phuUR0yp91-NdM6NfaStMXm8enJv2sKte6_HmCvstiy5a_RuXn52-EafKgGYO9KtyNKlnfk3LLl_IjIRRJY6ghy_6hzYszwAP1mDyw5Sn7q2ZDfnH8hQZBZXW-znFAxhXr-T4nbsus7PEEdEEYJk10aIX-HOoxxXAlA.Wo3JN-takALA-_h-Vw4dXA.3JRBTuabx2N5hfwU42QS3nk83xO_bnFqx3-x2ik2eW_GzTwQBb-2CZ2LTyKOHg-z_Hmr8ceVnPWexb56unfuJthoKCxwmwLrh2EPEBEMjpU.rt3Y5jkzKuFFwsF2OsiZDQ"
+        },
+    
+        "record": {
+            "CNAME": "imdakiki.github.io/earth"
```
Comment (CuteDog5695) [inline domains/earth.json:10]: Also please add more stuff to your website
```suggestion
            "CNAME": "imdakiki.github.io"
```

**Review by iostpa — DISMISSED** [review]

**Comment by imdakiki** [comment]
Done I added more content

**Comment by imdakiki** [comment]
anyone there

File: domains/qefw.json
Comment (orxngc) [inline domains/qefw.json:1]: This site is inaccessible, please provide a preview.

Comment (imdakiki) [reply domains/qefw.json:1]: i dont wanna do the qefw one

Comment (imdakiki) [reply domains/qefw.json:1]: can you do it now since i deleted the qefw file

**Pull request merged**